src/main/java/randoop/generation/GrtFuzzing.java

```diff
@@ -0,0 +1,480 @@
+        Sequence randomIndexSequence = Sequence.createSequenceForPrimitive(randomIndex);
+        return randomIndexSequence;
+      }
+
+      @Override
+      List<Executable> getMethods() throws NoSuchMethodException {
+        List<Executable> methodList = new ArrayList<>();
+        methodList.add(StringBuilder.class.getMethod("deleteCharAt", int.class));
+        methodList.add(StringBuilder.class.getMethod("toString"));
+        return methodList;
+      }
+    },
+
+    /** Replace a substring with a random character at a random index in the string. */
+    REPLACE {
+      @Override
+      Sequence getInputs(int stringLength) {
+        int randomIndex1 = Randomness.nextRandomInt(stringLength);
+        int randomIndex2 = Randomness.nextRandomInt(stringLength);
+        int startIndex = Math.min(randomIndex1, randomIndex2);
+        int endIndex = Math.max(randomIndex1, randomIndex2);
+        // ASCII 32-126
+        String randomChar = String.valueOf((char) (Randomness.nextRandomInt(95) + 32));
+        Sequence startIndexSequence = Sequence.createSequenceForPrimitive(startIndex);
+        Sequence endIndexSequence = Sequence.createSequenceForPrimitive(endIndex);
+        Sequence randomCharSequence = Sequence.createSequenceForPrimitive(randomChar);
+        return Sequence.concatenate(startIndexSequence, endIndexSequence, randomCharSequence);
+      }
+
+      @Override
+      List<Executable> getMethods() throws NoSuchMethodException {
+        List<Executable> methodList = new ArrayList<>();
+        methodList.add(
+            StringBuilder.class.getMethod("replace", int.class, int.class, String.class));
+        methodList.add(StringBuilder.class.getMethod("toString"));
+        return methodList;
+      }
+    },
+
+    /** Get a substring from a random index to another random index in the string. */
+    SUBSTRING {
+      @Override
+      Sequence getInputs(int stringLength) {
+        int randomIndex1 = Randomness.nextRandomInt(stringLength);
+        int randomIndex2 = Randomness.nextRandomInt(stringLength);
+        int startIndex = Math.min(randomIndex1, randomIndex2);
+        int endIndex = Math.max(randomIndex1, randomIndex2);
+        Sequence startIndexSequence = Sequence.createSequenceForPrimitive(startIndex);
+        Sequence endIndexSequence = Sequence.createSequenceForPrimitive(endIndex);
+        return Sequence.concatenate(startIndexSequence, endIndexSequence);
+      }
+
+      @Override
+      List<Executable> getMethods() throws NoSuchMethodException {
+        List<Executable> methodList = new ArrayList<>();
+        methodList.add(StringBuilder.class.getMethod("substring", int.class, int.class));
+        return methodList;
+      }
+    };
+
+    /** Get the inputs for the fuzzing operation as a sequence. */
+    abstract Sequence getInputs(int stringLength);
+
+    /** Get the methods for the fuzzing operation. */
+    abstract List<Executable> getMethods() throws NoSuchMethodException;
```

Comment:
I think a better name than `getMethods()` would be `get StringBuilderTransform()`, because it takes as input a StringBuilder and produces as output a StringBuilder.
Please also clarify the assumptions and relationship with `getInputs()`.  I think that it is assumed that no method in `getMethods()` takes any non-receiver arguments, except for exactly one method, which takes exactly `getInputs()` arguments.  Is that right?